Source/Models/BrickDimension.swift

```diff
@@ -138,6 +146,14 @@ public enum BrickDimension {
         }
     }
     
+    internal static func restrictedValue(for cellHeight: CGFloat, width: CGFloat, restrictedSize: RestrictedSize) -> CGFloat {
+        switch restrictedSize {
+        case .minimumSize(let minimumDimension):
+            return max(cellHeight, BrickDimension._rawValue(for: width, startingAt: 0, with: minimumDimension))
```

Comment:
We could do that but then the user of this would have to know that .auto cannot be used anywhere else